### PR TITLE
Yeni bir pr

### PR DESCRIPTION
**Pull request by Ece-Kahraman**
## Summary by CodeRabbit

- **Documentation**
	- Updated README with a humorous comment for a lighter tone.

### CODE REVIEW

**Comment by aysekelleci** [comment]
<pre><code class="language-markdown"># A first-level heading
## A second-level heading
### A third-level heading</code></pre>

**Comment by aysekelleci** [comment]
<h1>sdhkfhkdf</h1>

**Comment by aysekelleci** [comment]
<h1>sdhkfhkdf</h1>

**Comment by aysekelleci** [comment]
<blockquote><p>qewretrytjy</p></blockquote>

**Review by AlperMumcular — APPROVED** [review]

**Comment by vIremAydin** [comment]
**ACTIVE**:

File: README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ On systems lacking Adobe Acrobat, `qpdf` may be used to encrypt PDFs:
 for f in *.pdf; do qpdf --encrypt [pass] [pass] 256 -- "$f" "protected-$f"; done
 ```
```
Comment (AlperMumcular) [inline README.md:12]: a

**Review by AlperMumcular — CHANGES_REQUESTED** [review]
a

**Comment by aysekelleci** [comment]
> <blockquote><p>qewretrytjy</p></blockquote>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/12#issuecomment-2002484312 

okii

**Comment by aysekelleci** [comment]
> <blockquote><p>qewretrytjy</p></blockquote>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/12#issuecomment-2002484312 

gfhgfhfh

**Comment by aysekelleci** [comment]
**ACTIVE**: <p>selamlar</p>

**Pull request closed without merging**